pkg/NAMESPACE

```diff
@@ -181,5 +180,12 @@ export("cacheTable",
 
 export("sparkRSQL.init",
        "sparkRHive.init")
-export("print.structType",
-       "print.structField")
+
+export("structField",
+       "structField.jobj",
```

Comment:
It does in some cases. For example, when you call the `fields()` closure of a  pre-existing `structType` object, at that point you're trying to explicitly create a `structField` from a `jobj` directly and it won't be able to find the correct method:

```R
> testSchema$fields()
Error in UseMethod("structField", x) : 
 no applicable method for 'structField' applied to an object of class "jobj" 
```